### ISSUE TITLE
two-qubit Kraus map killing link

### ISSUE THREAD

**Issue by TysonRayJones** [QTechTheory/QuESTlink#76]
A single (i.e. a unitary) Kraus operator crashes QuESTlink.

MWE:

```MMA
m2[] := RandomVariate @ 
   CircularUnitaryMatrixDistribution[4]; 

UnitaryMatrixQ @ m2[]   (* True *)

rho = CreateDensityQureg[2];
ApplyCircuit[ Kraus_0,1[{ m2[] }], rho] (* kills link *)
```

**Comment by TysonRayJones** [comment]
Patched upstream (1e8dacc6a541d4f4d8a14c34acce57d7b7bd0cd8), not yet merged here

**Comment by TysonRayJones** [comment]
Merged in 248c0c45de0eec802dfa98336615e1e052381bb5
